Prace_domowe/Praca_domowa2/Grupa4/Mateusz Stączek/notebook.ipynb

```diff
@@ -0,0 +1,789 @@
+{
```

Comment:
<ul><li>Czemu spodziewałeś&nbsp;się&nbsp;innej liczby?</li><li>To prawda, można było użyć czegoś bardziej wyrafinowanego</li><li>Jakaś refleksja nt. Count Encoder?</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>